.github/workflows/fuzzer.yml

```diff
@@ -107,7 +117,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: qarmin/Qarminer
-          ref: ${{ env.GODOT_BASE_BRANCH }}
+          ref: 4.1
```

Comment:
Can you add a comment explaining why this is the chosen version?

Reply:
Done